test/github-issues/7110/entity/foo.entity.ts

```diff
@@ -9,7 +9,7 @@ export class Foo extends BaseEntity {
     @Column({
         nullable: false,
         type: "varchar",
-        default: () => "TO_CHAR(100, 'FMU000')",
+        default: () => "concat(chr(65), 'FMU000')",
```

Comment:
TO_CHAR() is not currently supported by cockorachdb, have to use a different function to test. This retains ::int and ::text parameters.